guides/common/modules/ref_kernel-parameters-for-discovery-customization.adoc

```diff
@@ -1,8 +1,7 @@
-[id="Unattended_Use_Customization_and_Image_Remastering_{context}"]
-= Unattended use, customization, and image remastering
+[id="kernel-parameters-for-discovery-customization_{context}"]
+= Kernel parameters for Discovery customization
 
-You can create a customized Discovery ISO to automate the image configuration process after booting.
-The Discovery image uses a Linux kernel for the operating system, which passes kernel parameters to configure the discovery service.
+Discovery uses a Linux kernel for the operating system, which passes kernel parameters to configure the Discovery service.
 These kernel parameters include the following entries:
 
 fdi.cachefacts::
```

Comment:
I would prefer to see this in a table instead of a def list. But it's your call. I think tables are easier to read and take up less space.

Reply:
Tables take much more space than a def list. I'm keeping it.